internal/api/v1/deploy.go

```diff
@@ -361,3 +365,11 @@ func existingReplica(ctx context.Context, client *k8s.Clientset, app models.AppR
 	}
 	return *result.Spec.Replicas, nil
 }
+
+func appDefaultRoute(ctx context.Context, name string) (string, error) {
```

Comment:
This can be moved to the domain package I think.

Reply:
will do

Reply:
Done. Commit b2a3b244de2d247b927ef74d1041a301a88498c3